app/multitenant/dynamo_collector.go

```diff
@@ -451,6 +573,22 @@ func (c *dynamoDBCollector) Add(ctx context.Context, rep report.Report) error {
 		return err
 	}
 
+	// fourth, put it in memcache
+	if c.memcache != nil {
+		// REVIEWER: Should we call this "Put" to be consistent with S3, or
+		// "Set" to be consistent with memcache?
```

Comment:
Ha I have no idea.  I don't think it really matters does it?  It will just come out in the graphs.